GitUI/CommandsDialogs/FormBrowse.cs

```diff
@@ -491,14 +491,32 @@ protected override void Dispose(bool disposing)
             {
                 // ReSharper disable ConstantConditionalAccessQualifier - these can be null if run from under the TranslatioApp
 
+                //// _buildReportTabPageExtension not disposable
+                //// _formBrowseDiagnosticsReporter not disposable
+                //// _submoduleStatusProvider is a singleton which must not be disposed
+                _windowsJumpListManager?.Dispose();
+                //// _aheadBehindDataProvider not disposable
+                //// _appTitleGenerator not disposable
+                //// _commitDataManager not disposable
+                //// _controller not disposable
                 _formBrowseMenus?.Dispose();
-                _filterRevisionsHelper?.Dispose();
                 _filterBranchHelper?.Dispose();
-                components?.Dispose();
+                _filterRevisionsHelper?.Dispose();
                 _gitStatusMonitor?.Dispose();
-                _windowsJumpListManager?.Dispose();
+                //// _splitterManager not disposable
```

Comment:
> Do we need all of these //// ... no disposable entries?

IMO yes, because GC does not work well for (NUnit) tests at least.
All fields (including the implicit ones created by properties) should be considered in every `Dispose` (in reverse order as created).